vignettes/epiparameter.Rmd

```diff
@@ -0,0 +1,521 @@
+  mean_sd = 0.1,
+  sd = convert_to_logsd(3, 1), 
+  sd_sd = 0.1, 
+  max = 10
+)
+
+reported_cases <- example_confirmed[1:40] 
+
+# use lnorm_logmeansd2meansd() to convert the parameters of the lognormal to the
+# mean (expectation) and standard deviation
+serial_interval_mean_sd <- lnorm_logmeansd2meansd(
+  meanlog = serial_interval$param[["meanlog"]], 
+  sdlog = serial_interval$param[["sdlog"]]
+)
+
+incubation_period_mean_sd <- lnorm_logmeansd2meansd(
+  meanlog = incubation_period$param[["meanlog"]], 
+  sdlog = incubation_period$param[["sdlog"]]
+)
+
+# for now serial interval is used in place of generation time
+generation_time_list <- list(
+  mean = serial_interval_mean_sd$mean
+  mean_sd = 0.1,
+  sd = serial_interval_mean_sd$sd,
+  sd_sd = 0.1,
+  max = 20
+) 
+
+incubation_period_list <- list(
+  mean = incubation_period_mean_sd$mean,
+  mean_sd = 0.1,
+  sd = incubation_period_mean_sd$sd,
+  sd_sd = 0.1,
+  max = 20
+)
+
+# estimate Rt by date of infection
+out <- epinow(
+  reported_cases = reported_cases, 
+  generation_time = generation_time_list,
+  rt = rt_opts(prior = list(mean = 2, sd = 0.1)),
+  delays = delay_opts(incubation_period_list, reporting_delay),
+  stan = stan_opts(samples = 750, chains = 4),
+  return_output = TRUE,
+  verbose = FALSE
+)
+```
+
+## Estimating $R_t$ using `R0`
+
+The R package `R0` contains a set of functions to estimate the basic reproduction number ($R_0$) and the effective reproduction number ($R_t$). Here we demonstrate a calculate of the time-varying $R_t$ using the method of @wallinga_different_2004. This method uses the epidemic curve of incidence data over the course of an epidemic and the generation time. 
+
+```{r generate generation time}
+# load data 
+data("Germany.1918")
+
+# generate generation times from discrete gamma distribution with mean 3 and 
+# standard deviation 1.5
+mGT <- generation.time("gamma", c(3, 1.5))
+```
+
+```{r, estimate Rt with R0}
+out <- estimate.R(
+  epid = Germany.1918, 
+  GT = mGT, 
+  begin = 1, 
+  end = 27, 
+  methods = "TD", 
+  nsim = 1e4
+)
+```
+
+```{r plot R0, eval=FALSE}
+plot(out)
+```
+
+```{r plotting function for R0, echo=FALSE, fig.align = "center", fig.height = 8, fig.width = 8, out.width = "500px", fig.cap="Estimated value of $R_t$ for the incidence data of Spanish flu for Germany 1918-1919."}
+
+plot_R0 <- function(out) {
+  plot(
+    as.Date(names(out$estimates$TD$R)), 
+    out$estimates$TD$R, 
+    ylim = c(0, max(out$estimates$TD$conf.int)), 
+    type = "l", 
+    xlab = "Date", 
+    ylab = "Rt"
+  )
+  
+  polygon(
+    x = c(rev(as.Date(rownames(out$estimates$TD$conf.int))), 
+          as.Date(rownames(out$estimates$TD$conf.int))), 
+    y = c(rev(as.vector(out$estimates$TD$conf.int["lower"])$lower),
+          as.vector(out$estimates$TD$conf.int["upper"])$upper), 
+    col = "grey80", 
+    border = NA
+  )
+  
+  lines(as.Date(names(out$estimates$TD$R)), out$estimates$TD$R)
+}
+
+plot_R0(out)
```

Comment:
I think it will be confusing for users if we rely on this trick of showing a chunk of code and evaluating another.

What don't you like in the default plotting method?